images/onnx-predictor-cpu/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN cd /tmp/ && \
 ENV S6_BEHAVIOUR_IF_STAGE2_FAILS 2
 
 RUN locale-gen en_US.UTF-8
-ENV LANG=en_US.UTF-8 LANGUAGE=en_US.en LC_ALL=en_US.UTF-8
+ENV LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8 LC_ALL=en_US.UTF-8 LC_MESSAGES=en_US.UTF-8
```

Comment:
In this case, would it be safer if we set the other locale variables as exemplified [here](https://unix.stackexchange.com/a/199306)? Ignore the SO question, just notice the output of `locale` in that case.

Reply:
Thanks for bringing this up! I confirmed that even without setting `LC_MESSAGES`, all of these variables are set properly (via `LC_ALL`), so I've removed it.